pkg/codegen/schema/schema.go

```diff
@@ -352,8 +352,10 @@ type Property struct {
 	Secret bool
 	// ReplaceOnChanges specifies if the property is to be replaced instead of updated (default false).
 	ReplaceOnChanges bool
-
-	Plain bool
+	// WillReplaceOnChanges indicates that the provider will replace the resource when
+	// this property is changed. This property is used exclusively for docs.
+	WillReplaceOnChanges bool
```

Comment:
Nit: given that this is docs-specific, I might have added this in a docs-specific entry in `Language` to more cleanly separate target-specific info from generic info. This does seem generic enough that I think it's fine where it is, even if it's only used by docs.